### ISSUE TITLE
Attach aria-labels

### ISSUE THREAD

**Issue by MatthewNobes** [MatthewNobes/MatthewNobes.github.io#39]
Aria-labels should be attached to every interactable item, and where appropriate, aria-descriptions/aria-describeby should also be used. All of this should aid the accessibility of the site.
